bpf/bpf_lxc.c

```diff
@@ -155,6 +155,11 @@ ipv6_l3_from_lxc(struct __ctx_buff *ctx, struct ipv6_ct_tuple *tuple,
 		 */
 		svc = lb6_lookup_service(&key, is_defined(ENABLE_NODEPORT));
 		if (svc) {
+			if (lb6_svc_is_l7loadbalancer(svc)) {
+				verdict = svc->l7_lb_proxy_port;
```

Comment:
'verdict' normally carries the proxy port from the policy enforcement code (if any), so here we reuse `verdict` the same way it is used elsewhere. Later commits make this a bit clearer, I hope:
```
		/* Restore data from service handling. */
#if defined(ENABLE_L7_LB)
		verdict = ctx_load_meta(ctx, CB_PROXY_MAGIC) >> 16;
		if (verdict != 0) {
			proxy_port = (__u16)verdict;
			/* tuple addresses have been swapped by CT lookup */
			cilium_dbg3(ctx, DBG_L7_LB, tuple.daddr.p4, tuple.saddr.p4,
				    bpf_ntohs(proxy_port));
			ctx_store_meta(ctx, CB_PROXY_MAGIC, 0);
			goto skip_policy_enforcement;
		}
#endif /* ENABLE_L7_LB */
```
Even in this case the code after `skip_policy_enforcement` label will use the `verdict` variable like it needs to if we did not skip policy enforcement. Here passing the proxy port in `verdict` is a method to share the reuse the redirection logic that is otherwise used for L7 egress policy enforcement.